lib/model.php

```diff
@@ -76,10 +80,18 @@ public function purge()
      */
     private function _getStore()
     {
+        // FIXME
+        // Workaround so that config value don't need to be changed
```

Comment:
We need such a workaround, but it should be done in the configuration class. Please change the default [here](https://github.com/Sobak/PrivateBin/blob/bb58b4bf85300c8a5b0eb88cfd70c4fa8c7c1b2c/lib/configuration.php#L85) and add the replacements in [here](https://github.com/Sobak/PrivateBin/blob/bb58b4bf85300c8a5b0eb88cfd70c4fa8c7c1b2c/lib/configuration.php#L199).

Side note: One of the reasons why compatibility to old Zerobin is so important to me is that I run an instance that was upgraded since the first 0.15 release in 2012. And I want to keep certain paste URLs to work, including some long running comment threads.